nilearn/reporting/glm_reporter.py

```diff
@@ -302,41 +312,6 @@ def _check_report_dims(report_size):
     return width, height
 
 
-def _coerce_to_dict(input_arg):
```

Comment:
extracted into nilearn/reporting/utils to be able to import into the experimental module and avoid circular import